src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,15 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
+        User userByEmail = userService.findByEmail(email);
+
+        if (userByEmail != null) {
+            if (userByEmail.getEmail().equals(email)
+                    && userByEmail.getPassword().equals(password)) {
+                return true;
```

Comment:
you don`t need to check email cause you get this user by email already

Reply:
make sense!